app/views/conversations/_conversation.haml

```diff
@@ -16,10 +16,10 @@
               = other_participants.count - 1
 
       .bg
-        .badge.badge-dafault.message_count
+        .badge.badge-default.message_count
           = conversation.messages.size
         - if visibility.unread > 0
-          .badge.badge-important.unread_message_count
+          .badge.progress-bar-danger.unread_message_count
```

Comment:
I don't think we should misuse Bootstrap classes. Instead you could set a red background for `.badge-important` via CSS.